### ISSUE TITLE
No softmax activation for inverse net

### ISSUE THREAD

**Issue by bonniesjli** [jcwleo/curiosity-driven-exploration-pytorch#5]
Hi! For the inverse net in the ICM model, you did not use a SoftMax activation function. 
But according to the paper:  
In case a_t (action) is discrete, the output of g is a soft-max distribution across all possible actions.

**Comment by jcwleo** [comment]
You can find softmax in here(https://github.com/jcwleo/curiosity-driven-exploration-pytorch/blob/master/agents.py#L59)
